include/lsst/afw/table/Schema.h

```diff
@@ -171,7 +184,7 @@ class Schema {
      *  @brief Add a new field to the Schema, and return the associated Key.
      *
      *  This is simply a convenience wrapper, equivalent to:
-     *  @code
+     *  @codeinspect
```

Comment:
Looks like I thought I was typing into another window and committed it.  I'll revert this entirely.